decs.d.ts

```diff
@@ -14,3 +14,7 @@ declare module "@jsonforms/core" {
   const and: any;
   const uiTypeIs: any;
 }
+
+declare module "@material-ui/core" {
+  const Typography: any;
+}

```

Comment:
It is very surprising that there's no type coverage for @material-ui components... can this be validated?

Reply:
I think the types don't work because material upgraded and deprecated v4. I've added @material-ui/core as a dev dependency and that worked, is that alright?